google/cloud/internal/oauth2_minimal_iam_credentials_rest_test.cc

```diff
@@ -268,6 +275,16 @@ TEST(MinimalIamCredentialsRestTest, GenerateAccessTokenSuccess) {
   EXPECT_THAT(access_token->token, Eq("my_access_token"));
 }
 
+TEST(MinimalIamCredentialsRestTest, GenerateAccessTokenSuccess) {
+  TestGenerateAccessTokenWithUniverseDomain(absl::nullopt);
```

Comment:
I am not a fan of adding extra logic in unit tests. I think it is easier to read if we spell out the code twice, instead of trying to factor out commonalities.

Reply:
Done.